test/hotspot/gtest/metaspace/test_virtualspacenode.cpp

```diff
@@ -1,6 +1,6 @@
 /*
- * Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
- * Copyright (c) 2020 SAP SE. All rights reserved.
+ * Copyright (c) 2020, 2021, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2020, 2021 SAP SE. All rights reserved.
```

Comment:
There should add a ',' after 2021

Reply:
Thanks, Yumin! Sorry, I already had integrated under the trivial rule. And yes, Coleen is right, we do not have that trailing comma (don't ask me why).

..Thomas